configserver/src/main/java/com/yahoo/vespa/config/server/session/PrepareParams.java

```diff
@@ -253,6 +272,53 @@ public static PrepareParams fromHttpRequest(HttpRequest request, TenantName tena
                             .build();
     }
 
+    public static PrepareParams fromJson(byte[] json, TenantName tenant, Duration barrierTimeout) {
+        Slime slime = SlimeUtils.jsonToSlime(json);
+        Inspector params = slime.get();
+
+        return new Builder().ignoreValidationErrors(params.field(IGNORE_VALIDATION_PARAM_NAME).asBool())
```

Comment:
Won't this fail if there is no field `IGNORE_VALIDATION_PARAM_NAME`?   Check with `valid()` first?

Reply:
Sure, that was just an example, though, same issue with other fields that are not checked explicitly

Reply:
Yes. Looks to me like all bool fields are OK, but I didn't check the ones that have separate serialisers.